### ISSUE TITLE
Doesn't work with Terraform 0.12

### ISSUE THREAD

**Issue by Dzhuneyt** [terraform-community-modules/tf_aws_nat#22]
I get the following error:

```
Error: Unsupported block type

  on .terraform/modules/nat/main.tf line 31, in data "template_file" "user_data":
  31:   vars {

Blocks of type "vars" are not expected here. Did you mean to define argument
"vars"? If so, use the equals sign to assign it a value.

```

**Comment by mgfinch** [comment]
exactly my problem... will be solved by accepting #21

**Comment by antonbabenko** [comment]
#21 has been merged and [v1.4.0](https://github.com/terraform-community-modules/tf_aws_nat/releases/v1.4.0) was just released.
